tests/ops/api/v1/endpoints/test_system.py

```diff
@@ -33,7 +38,37 @@ def url_invalid_system() -> str:
     return V1_URL_PREFIX + f"/system/not-a-real-system/connection"
 
 
-class TestPatchSystemConnections(TestPatchConnections):
+@pytest.fixture(scope="function")
+def payload():
+    return [
+        {
+            "name": "My Main Postgres DB",
+            "key": "postgres_db_1",
+            "connection_type": "postgres",
+            "access": "write",
+            "secrets": {
+                "url": None,
+                "host": "http://localhost",
+                "port": 5432,
+                "dbname": "test",
+                "db_schema": "test",
+                "username": "test",
+                "password": "test",
+            },
+            "enabled_actions": ["access", "erasure"],
```

Comment:
do we actually ever validate that this is getting saved to the connection config?

it doesn't need to be in this test module specifically, but it'd be nice to get some validation that this property (as well as some of the other properties we're passing in the payload here) are actually correctly populated. essentially, there's a fair amount of logic in `connection_util.patch_connection_configs` that i think it deserves some more test coverage than we have currently (as far as i can tell).

it's not that you've changed that function so much in this PR, it's more just that it feels like that's potentially a blind spot in some of our test coverage, and now that's becoming more evident as i look to see whether we're validating that your updates here to include `enabled_actions` actually get persisted. i wouldn't consider this a blocker for this PR, but i'm a bit worried about that lack of test coverage on an important piece of code that's not so straightforward.

maybe i've missed some of the test coverage though and i'm making noise about nothing! 😄